udp_server.go

```diff
@@ -257,9 +257,27 @@ func udpServer() {
 			log.Printf("received: %s from %s\n", sdata, remote)
 		}
 
+		// Check if the message has a key named "type" and rename it to "read_type"
+		if val, ok := packet["type"]; ok {
+			packet["read_type"] = val
+			delete(packet, "type")
+		}
+
 		// send data to Stomp endpoint
 		if Config.Endpoint != "" && stompConn != nil {
-			sendDataToStomp(data)
+			newData, err := json.Marshal(packet)
+			if err != nil {
+				log.Printf("unable to marshal UDP packet into JSON, error %v\n", err)
+				// clear-up our buffer
+				buffer = buffer[:0]
+				continue
+			}
+			if Config.Verbose {
+				sNewData := strings.TrimSpace(string(newData))
+				log.Printf("sent to AMQ: %s\n", sNewData)
+			}
+			log.Printf("sent: %s \n", newData)
```

Comment:
Please do not dump data to the log as your log will grow very fast. Check the rate of the data, I think we get millions of docs and in this case you will fill out your log in a day or two on local file system. Instead, put this printout only in verbose mode.